CONTRIBUTING.md

```diff
@@ -132,8 +155,8 @@ tools:
 -  No PEP8 warnings, check with:
 
   ```bash
-  $ pip install pep8
-  $ pep8 path/to/module.py
+  $ pip install pycodestyle  # formerly called pep8 
+  $ pycodestyle path/to/module.py
```

Comment:
A small thing, and occurs throughout (perhaps intentionally), but the presence of the `$`s at the beginning of these lines makes it awkward to copy and paste the commands. I found the same when I was exercising the "system info-gathering tools" when submitting an issue.

I also wonder whether there couldn't be an option for contributors to run a more streamlined command of some sort that would perform all checks. For example, in my personal projects, I've enjoyed having pytest use the pytest-flake8 and pytest-black (for example) modules to ensure that style is maintained. 

Perhaps some sort of auxiliary file (that could be part of the official build) that optionally ensures requirements are present and then runs the tests/checks? The more I type, the more I believe this is a larger issue…

Reply:
Not sure about this one. This seems to be the common style use for command line code, but I see your point. Referring this to @darcymason and @scaramallion...

Reply:
> Perhaps some sort of auxiliary file (that could be part of the official build) that optionally ensures requirements are present and then runs the tests/checks? The more I type, the more I believe this is a larger issue…

I set up my own git to run flake8 and pytest before committing (and aborting commit if errors occur).  However, it was actually quite complicated and I couldn't get it working until I found a blog post with a detailed script -- can't remember if it was Windows-specific problem or not.  I'll see if I can find again what I did and the script.  It's not a big deal, since both things are caught when the github checks are run anyway.


> his seems to be the common style use for command line code, but I see your point. Referring this to @darcymason and @scaramallion...

Seems to be my day to be not very decisive.  It doesn't really matter to me.  I don't think it is a great hardship to copy one line at a time.  However I could see that the "system info gathering" script in the issue template should be easy to copy and paste and get results.